server/src/core/sentence-collector/validation/languages/en.ts

```diff
@@ -0,0 +1,41 @@
+import { ValidatorRule } from '../../types/validators'
+
+const tokenizeWords = require('talisman/tokenizers/words')
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 14
+
+const INVALIDATIONS: ValidatorRule[] = [
+  {
+    type: 'fn',
+    fn: (sentence: string) => {
+      const words = tokenizeWords(sentence)
+      return words.length < MIN_WORDS || words.length > MAX_WORDS
+    },
+    error: `Number of words must be between ${MIN_WORDS} and ${MAX_WORDS} (inclusive)`,
+  },
+  {
+    type: 'regex',
+    regex: /[0-9]+/,
+    error: 'Sentence should not contain numbers',
```

Comment:
### Question

I just wanted to ask why a sentence can't have numbers. Is a sentence like:

> She has 2 cats

not valid because of the number "2"?

Reply:
Yes, that would mean that the sentence
> She has 2 cats

would be rejected from submission. I am not entirely sure why that rule is necessary, though. Maybe @ftyers can help.

Reply:
We have had that from the beginning basically. The reasoning back then was that for non-single-digit numbers there can be quite some ambiguity and unclarity on how to read it. For example 2020 could be read as:

- two zero two zero
- two thousand and twenty
- twenty twenty

I'm not sure how much sense this still makes, but that was the reasoning back then.

Reply:
Thanks for the explanation 👍